scripts/test-util/1.sh

```diff
@@ -1,8 +1,8 @@
 ### IBC TRANSFER
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
If you intended to commit the test-util/N.sh scripts, could you please add a comment in the PR description as to why?

Reply:
Discussed offline - going to keep scripts 1-4 but remove scratch files